src/config.rs

```diff
@@ -153,6 +156,7 @@ impl ConfigBuilder {
             request_response_protocols: HashMap::new(),
             known_addresses: Vec::new(),
             connection_limits: ConnectionLimitsConfig::default(),
+            keep_alive_timeout: Duration::from_secs(5),
```

Comment:
Could you put this constant into [`transport/mod.rs`](https://github.com/paritytech/litep2p/blob/aa18af8fd4d2209e849308df08b4c792e6a468db/src/transport/mod.rs#L44-L48) with other timeouts, and use it (including tests) instead of hardcoding `Duration::from_secs(5)`, please?